src/networks/networks.jl

```diff
@@ -129,11 +129,12 @@ end
 add_labels!(tensors::AbstractVector{<:AbstractArray}, ixs, labels) = tensors
 
 const SetPolyNumbers{T} = Union{Polynomial{T}, TruncatedPoly{K,T} where K, CountingTropical{TV,T} where TV} where T<:AbstractSetNumber
-function add_labels!(tensors::AbstractVector{<:AbstractArray{T}}, ixs, labels) where T <: Union{AbstractSetNumber, SetPolyNumbers, ExtendedTropical{K,T} where {K,T<:SetPolyNumbers}}
+function add_labels!(tensors::AbstractVector{<:AbstractArray{T}}, ixs, labels, fix_config=Dict()) where T <: Union{AbstractSetNumber, SetPolyNumbers, ExtendedTropical{K,T} where {K,T<:SetPolyNumbers}}
```

Comment:
A recommended approach to do this is
```julia
select_dims(add_labels!(tensors, ixs, labels), ixs, fixedconfigs)
```

Then you do not need to modify this function.